apps/src/EditorAnnotator.js

```diff
@@ -1,11 +1,12 @@
 import md5 from 'md5';
-import RGBColor from 'rgbcolor';
 
 import annotationList from '@cdo/apps/acemode/annotationList';
 import {singleton as studioApp} from '@cdo/apps/StudioApp';
 import {border_gray} from '@cdo/apps/util/color';
 import {interpolateColors} from '@cdo/apps/utils';
 
+import RGBColor from './lib/util/rgbcolor.js';
```

Comment:
My first thought here is that maybe we should be going the other direction: removing out internal "fork" of rgbcolor in favor of the npm? Do we know what changes we made to rgbcolor.js? In general, we are not pushing hard to remove dependencies, esp when they are from older parts of our codebase. There's a significant risk in each change, and not a huge upside.